### PR TITLE
data: dfdevaney

### PR DESCRIPTION
**Pull request by dfdevaney**
## Fixes Issue

Edited a dead icon.

## Changes proposed

Added some links. 

## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ x] My change requires changes to the documentation.
- [ x] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thanks for looking it over!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8610"></a>

### CODE REVIEW

File: data/dfdevaney.json [hunk 1]
```diff
@@ -15,7 +15,17 @@
     {
       "name": "LinkedIn",
       "url": "https://www.linkedin.com/in/doug-devaney",
-      "icon": "FaLinkdedIn"
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Threads",
+      "url": "https://www.threads.net/@Snoops127",
+      "icon": "FaThreads"
```
Comment (sandramsc) [inline data/dfdevaney.json:23]: Hi, "FaThreads" is not a valid icon. Consider searching for available icons [here](https://linkfree.io/icons)
icon

```suggestion

```

**Review by sandramsc — CHANGES_REQUESTED** [review]

File: data/dfdevaney.json (hunk 1 above)
Comment (dfdevaney) [reply data/dfdevaney.json:23]: Thanks!
I was wondering where the icon list was located.
I will need to make an adjustment later.

File: data/dfdevaney.json [hunk 2]
```diff
@@ -15,7 +15,17 @@
     {
       "name": "LinkedIn",
       "url": "https://www.linkedin.com/in/doug-devaney",
-      "icon": "FaLinkdedIn"
+      "icon": "FaLinkedIn"
```
Comment (sital002) [inline data/dfdevaney.json:18]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by sital002 — CHANGES_REQUESTED** [review]
FaLinkedIn isnot valid. It should be FaLinkedin or FaLinkedinIn.

File: data/dfdevaney.json (hunk 1 above)
Comment (SaraJaoude) [reply data/dfdevaney.json:23]: Icons not found will go to a default icon. It is best to keep the property and not delete this icon line altogether.

I will merge this PR and once you wish to make the adjustment @dfdevaney please raise a new PR.

**Pull request merged**